theoads/getting-started/00-getting-started-signaling-service.mdx

```diff
@@ -176,9 +222,9 @@ The response is an array of the existing monitized stream resources.
       "labels": [],
       "layout": "DOUBLE",
       "origin": "https://domain.com",
+      "segmentOrigin": "https://segment-domain.com",
       "assetKey": "google-sgai-demo",
       "streamId": "theoads-demo",
-      "originPath": "/",
```

Comment:
OriginPath is not used in our signaling server and was also not specified in the parameters above so got rid of it.